pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1224,12 +1226,19 @@ func setupDomainMemory(vmi *v1.VirtualMachineInstance, domain *api.Domain) error
 		return err
 	}
 
+	blockAlignment := MemoryHotplugBlockAlignmentBytes
+	if vmi.Spec.Domain.Memory != nil &&
+		vmi.Spec.Domain.Memory.Hugepages != nil &&
+		vmi.Spec.Domain.Memory.Hugepages.PageSize == "1Gi" {
+		blockAlignment = MemoryHotplug1GHugePagesBlockAlignmentBytes
+	}
+
 	domain.Spec.Devices.Memory = &api.MemoryDevice{
 		Model: "virtio-mem",
 		Target: &api.MemoryTarget{
 			Size:  pluggableMemorySize,
 			Node:  "0",
-			Block: api.Memory{Unit: "b", Value: MemoryHotplugBlockAlignmentBytes},
+			Block: api.Memory{Unit: "b", Value: uint64(blockAlignment)},
```

Comment:
Nit: can't we just declare the consts as uint64?

Reply:
Yes but then it gets used in other places, see https://github.com/search?q=repo%3Akubevirt%2Fkubevirt%20MemoryHotplugBlockAlignmentBytes&type=code.

So I preferred to have less casts globally :)